shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/slf4j/DeployedLogger.java

```diff
@@ -34,6 +36,19 @@ public void logInfo(String infoMessage, Object... parameters) {
         logBuilder.log();
     }
 
+    @Override
+    public void logMap(String baseMessage, Map<String, Object> map) {
+        Level level = Level.INFO;
+        var logBuilder = LoggerHelper.logMessageAtLevel(LOGGER, level, baseMessage);
+
+        logMapFields(logBuilder, map);
+        logBuilder.log();
+    }
+
+    private void logMapFields(LoggingEventBuilder logger, Map<String, Object> map) {
+        map.forEach(logger::addKeyValue);
+    }
```

Comment:
What do you think about removing this method since it's only one line?  We can just move `map.forEach(logger::addKeyValue);` into the `logMap` method.  🤷🏻‍♂️

Reply:
done.